### PR TITLE
[Sale] ADD: security group for analytic tags

### PR DESCRIPTION
**Pull request by Yenthe666**
Description of the issue/feature this PR addresses: The analytic accounting tags are shown in the sale view but are filtered out anywhere else thanks to a security group.

Current behavior before PR: The analytic accounting tags are shown in the sale view but are filtered out anywhere else thanks to a security group.

Desired behavior after PR is merged: Other references to the field in sale.order.line uses the access group group_analytic_accounting. This logic is now extended to this view too. 
Fixes https://github.com/odoo/odoo/issues/18875

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Yenthe666** [comment]
@qdp-odoo this is the fix for https://github.com/odoo/odoo/issues/18875

File: addons/sale/views/sale_views.xml [hunk 1]
```diff
@@ -243,7 +243,7 @@
                                             </div>
                                             <label for="analytic_tag_ids"/>
```
Comment (wtaferner) [inline addons/sale/views/sale_views.xml]: Shouldn't this label also be hidden?

Comment (qdp-odoo) [inline addons/sale/views/sale_views.xml]: yes indeed.

Comment (Yenthe666) [inline addons/sale/views/sale_views.xml]: Good catch - see the second commit.

**Pull request closed without merging**